samples/ko/Getting Started/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html>
+    
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>BRACKETS으로 시작하기</title>
```

Comment:
I don't know korean language, is it possible to add a space after BRACKETS? Here and below.

Reply:
@ficristo

Reply:
Yes. I don't know if there is a grammar rule that forbid it, but otherwise I think it would look nicer with a whitespace.

Reply:
@ficristo  Okay, I will fix it.

Reply:
@ficristo 

I fixed all your comments.
Please confirm.